src/main/java/org/opensearch/security/dlic/rest/api/CertificatesApiAction.java

```diff
@@ -0,0 +1,120 @@
+        PLUGIN_API_ROUTE_PREFIX
+    );
+
+    public CertificatesApiAction(
+        final ClusterService clusterService,
+        final ThreadPool threadPool,
+        final SecurityApiDependencies securityApiDependencies
+    ) {
+        super(Endpoint.SSL, clusterService, threadPool, securityApiDependencies);
+        this.requestHandlersBuilder.configureRequestHandlers(this::securitySSLCertsRequestHandlers);
+    }
+
+    @Override
+    public List<Route> routes() {
+        return ROUTES;
+    }
+
+    @Override
+    public String getName() {
+        return "HTTP and Transport Certificates Actions";
+    }
+
+    @Override
+    protected CType getConfigType() {
+        return null;
+    }
+
+    @Override
+    protected void consumeParameters(RestRequest request) {
+        request.param("nodeId");
+        request.param("cert_type");
+    }
+
+    private void securitySSLCertsRequestHandlers(RequestHandler.RequestHandlersBuilder requestHandlersBuilder) {
+        requestHandlersBuilder.withAccessHandler(this::accessHandler)
+            .allMethodsNotImplemented()
+            .verifyAccessForAllMethods()
+            .override(
+                RestRequest.Method.GET,
+                (channel, request, client) -> client.execute(
+                    CertificatesActionType.INSTANCE,
+                    new CertificatesInfoNodesRequest(
+                        CertificateType.from(request.param("cert_type")),
+                        true,
+                        request.paramAsStringArrayOrEmptyIfAll("nodeId")
+                    ).timeout(request.param("timeout")),
+                    new ActionListener<>() {
+                        @Override
+                        public void onResponse(final CertificatesNodesResponse response) {
+                            ok(channel, (builder, params) -> {
+                                builder.startObject();
+                                RestActions.buildNodesHeader(builder, channel.request(), response);
+                                builder.field("cluster_name", response.getClusterName().value());
+                                response.toXContent(builder, channel.request());
+                                builder.endObject();
+                                return builder;
+                            });
+                        }
+
+                        @Override
+                        public void onFailure(Exception e) {
+                            LOGGER.error("Cannot load SSL certificates info due to", e);
+                            internalServerError(channel, "Cannot load SSL certificates info " + e.getMessage() + ".");
+                        }
+                    }
+                )
+            );
+    }
+
+    boolean accessHandler(final RestRequest request) {
+        if (request.method() == RestRequest.Method.GET) {
```

Comment:
Is it necessary to check the method here? When defining the handler, I see that it uses `.allMethodsNotImplemented()`. If a user tried to use a verb other than get, would they get a not implemented error?

Reply:
`allMethodsNotImplemented` - add not impleneted handle for all methods which the sec plugin supports (I took them as is from existing solution).   `accessHandler` I just moved as is from `SecuritySSLCertsApiAction` so it looks the same. I will add `PUT` later